archive/archive.go

```diff
@@ -364,16 +364,13 @@ func copyDirWalkFn(
 
 			// Copy the concrete entry for this path. This will either
 			// be the file itself or just a directory entry.
-			if err := copyConcreteEntry(tarW, subpath, target, info); err != nil {
-				return err
-			}
+			return copyConcreteEntry(tarW, subpath, target, info)
```

Comment:
ah shoot you're right.  I'll work on the dir case. ping you again when this is ready :sweat_smile: